client/src/components/pages/application/Accounts/index.css

```diff
@@ -20,7 +20,7 @@
 .Form_container {
   position: absolute;
   top: 20%;
-  right: 35%;
+  right: 20%;
```

Comment:
why you change this ?? keep it as before 
```suggestion
  right: 35%;
```

Reply:
### **If I change it to 35%, it will become like this picture**
![Screenshot from 2020-06-03 17-41-30](https://user-images.githubusercontent.com/49004640/83651579-51a18600-a5c2-11ea-8431-dfbbb29363e0.png)
### **But I can change it to 30% to be like this**
![Screenshot from 2020-06-03 17-40-46](https://user-images.githubusercontent.com/49004640/83652119-ec9a6000-a5c2-11ea-9e22-5be585c112cf.png)